core/src/test/java/edu/wpi/grip/util/GRIPCoreTestModule.java

```diff
@@ -69,6 +74,12 @@ public void tearDown() {
     protected void configure() {
         assert setUp : "The GRIPCoreTestModule handler was not set up. Call 'setUp' before passing the injector";
         super.configure();
+        // HTTP server injection bindings
+        bind(GripServer.HttpServerFactory.class).to(GripServerTest.TestServerFactory.class);
+        bind(GripServer.class).asEagerSingleton();
+        bind(MapNetworkPublisherFactory.class)
+                .annotatedWith(Names.named("httpManager"))
+                .to(HttpManager.class);
```

Comment:
Every test class that uses injection will fail because the injector can't be created due to no implementation bound for HttpServerFactory when it's trying to create an HttpSource. So this has to be in here because the source factories are bound in `GRIPCoreModule.java`

[Stacktrace](https://gist.github.com/SamCarlberg/bd9ae83998b1d283df69)